onnx/common/ir.h

```diff
@@ -73,6 +73,7 @@ struct Dimension final {
   Dimension(std::string param)
     : is_int(false), dim(-1), param(std::move(param)) {
   }
+  Dimension(int64_t dim) : is_int(true), dim(dim) {}
```

Comment:
@zrphercule @anderspapitto Should we change the original version to take `int64_t`? I think then the compiler won't complain for neither passing in `int` nor `int64_t`.

Reply:
I guess you are right. I see no other use of this "int" version, but I dunno if there is any.

Reply:
Yeah, I also vote for `int64_t`. We use `int64_t` for dimensions in onnx proto, so it would be nice to keep consistency.

Reply:
sounds good to me